README.md

```diff
@@ -1,82 +1,23 @@
 # GrimoireLab Tutorial
 
-[GrimoireLab](http://grimoirelab.github.io/) is free, open source software for software development analytics. It allows you to retrieve data from many kinds of systems with information related to software development, and produce analysis and visualizations with it.
+[GrimoireLab](http://grimoirelab.github.io/) is free, open source software for software
+development analytics. It allows you to retrieve data from many kinds of systems with
+information related to software development, and produce analysis and visualizations with
+it.
 
 ![](assets/wizard-y.svg)
```

Comment:
I think we can slowly update the images, I have created some graphics using existing images.

![](https://raw.githubusercontent.com/vchrombie/misc/master/chaoss/grimoirelab-chaoss.png)

![](https://raw.githubusercontent.com/vchrombie/misc/master/chaoss/grimoirelab-3.png)

![](https://raw.githubusercontent.com/vchrombie/misc/master/chaoss/grimoirelab-2.png)

We need not add them in this PR, but I'm just putting a comment over here.